demisto_sdk/commands/common/errors.py

```diff
@@ -2547,3 +2550,18 @@ def modeling_rule_keys_are_missing():
     def invalid_rule_name(invalid_files):
         return f"The following rule file name is invalid {invalid_files} - make sure that the rule name is " \
                f"the same as the folder containing it."
+
+    @staticmethod
+    @error_code_decorator
+    def modeling_rule_missing_testdata_file(modeling_rule_dir: Path, minimum_from_version: str, from_version: str):
+        test_data_file = modeling_rule_dir / f'{modeling_rule_dir.name}_testdata.json'
+        return f'The modeling rule {modeling_rule_dir} is missing a testdata file at {test_data_file}. The modeling ' \
+               f'rule has a fromversion of {from_version} and modeling rules supporting fromversion' \
+               f' {minimum_from_version} and upwards are required to provide test data for modeling rule testing.' \
+               ' Please add a testdata file for this rule. See the "demisto-sdk modeling-rules init-test-data" ' \
+               ' command for more information on creating a test data file for modeling rules.'
```

Comment:
2 spaces...
In general - Can you make the lines standard?
Decide whether the space should be at the end or beginning of lines.
```suggestion
               'command for more information on creating a test data file for modeling rules.'
```